### PR TITLE
Add instructions for installing via AppImage

### PR DESCRIPTION
**Pull request by gautampanchal94**
Instructions for Linux Systems using AppImage. (Con. of PR:#4866)

### CODE REVIEW

**Comment by David-Else** [comment]
Have you tested the auto-update? I don't know how it works, but it seems it might involve:

- using helix-22.12-x86_64.AppImage.zsync 
- https://github.com/AppImageCommunity/AppImageUpdate
- https://github.com/AppImageCommunity/zsync2
 
?

If so, would be great to get that documented... I have not tried myself.

File: book/src/install.md [hunk 1]
```diff
@@ -87,6 +88,16 @@ accepts the new settings on first use.
...
 [install Cachix CLI](https://docs.cachix.org/installation) and use
 `cachix use helix` to configure Nix to use cached outputs when possible.
+
+### AppImage
+
+Install Helix using [AppImage](https://appimage.org/).
+Download Helix AppImage from the [Releases](https://github.com/helix-editor/helix/releases) page.
+
+```sh
+chmod +x helix-*.AppImage # change permission for exercutable mode
```
Comment (the-mikedavis) [inline book/src/install.md:98]:
```suggestion
chmod +x helix-*.AppImage # change permission for executable mode
```

File: book/src/install.md [hunk 2]
```diff
@@ -87,6 +88,16 @@ accepts the new settings on first use.
 If you are using a version of Nix without flakes enabled,
 [install Cachix CLI](https://docs.cachix.org/installation) and use
 `cachix use helix` to configure Nix to use cached outputs when possible.
+
+### AppImage
+
+Install Helix using [AppImage](https://appimage.org/).
+Download Helix AppImage from the [Releases](https://github.com/helix-editor/helix/releases) page.
```
Comment (the-mikedavis) [inline book/src/install.md:95]:
```suggestion
Download Helix AppImage from the [latest release](https://github.com/helix-editor/helix/releases/latest) page.
```

We can point to the latest release with that special `/latest` URL

Comment (gautampanchal94) [reply book/src/install.md:95]: Ok

**Review by the-mikedavis — APPROVED** [review]

**Pull request merged**